tests/integration/polygonParts/polygonParts.spec.ts

```diff
@@ -4846,6 +4846,121 @@ describe('polygonParts', () => {
         expect.assertions(29);
       });
 
+      it('should return 201 status code and create the resources for multiple parts, where polygon parts are not generated for a small area generated by splitting other incoming part', async () => {
```

Comment:
please make it more understandable so desc will define the test case that leaving an small polygon which is removed

Reply:
changed